langlib/langlib-test/src/test/resources/test-src/arraylib_test_negative.bal

```diff
@@ -0,0 +1,70 @@
+function testPushOnFixedLengthArray() {
```

Comment:
Shall we add the licence header for the bal file too?

Reply:
Test cases dont need license header right.

Reply:
We've added the licence header in most of the new tests we've added. And given that the Java class for a test always has the licence header, I think we should include it in the bal file too.

Reply:
added by 35e88fad66625c0dd4c8d99af1fa2ec33ec85c1d